lib/core/application/event/event_detail_cohosts_bloc/event_detail_cohosts_bloc.dart

```diff
@@ -19,6 +19,7 @@ class EventDetailCohostsBloc
     EventDetailCohostsEventFetch event,
     Emitter emit,
   ) async {
+    emit(const EventDetailCohostsState.loading());
```

Comment:
The name of this bloc is really hard to understand what it's trying to do, should we rename GetEventCohostRequestsBloc ?